### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by vikabelous**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # check-notifications
 
-fancy a pr
+bul' bul' bul'
```
Comment (kudrykv) [inline README.md:3]: testie test test

**Comment by kudrykv** [comment]
another comment

**Review by kudrykv — COMMENTED** [review]
comment

File: README.md (hunk 1 above)
Comment (vikabelous) [reply README.md:3]: bloba?

Comment (kudrykv) [reply README.md:3]: bloba!

**Comment by vikabelous** [comment]
he-he

**Pull request merged**